smart-client-core/src/main/java/com/emc/rest/smart/SmartException.java

```diff
@@ -0,0 +1,31 @@
+package com.emc.rest.smart;
+
+public class SmartException extends RuntimeException {
```

Comment:
Does it look better if rename the Exception to sth like SmartClientException or SmartClientHandlerException?